MBBSEmu/DOS/Interrupts/Int21h.cs

```diff
@@ -166,6 +172,112 @@ public void Handle()
             }
         }
 
+        private void ClearCarryFlag() => _registers.F = _registers.F.ClearFlag((ushort)EnumFlags.CF);
```

Comment:
Add the compiler optimization for `Subroutine` optimization so this gets inlined to avoid the additional call.